apps/condo/migrations/20220215172458-00100_alter_user_type.js

```diff
@@ -0,0 +1,24 @@
+// auto generated by kmigrator
```

Comment:
are we sure we really need that migration

Reply:
yes, it's because of the new type. 
But you can add `// KMIGRATOR:0100_a...` comment to the last master migration to avid redundant new file

Reply:
Yes, it doesn't works without this empty migration

Reply:
yes, but it's a kmigrotor logic %) it's a Django way